dev-games/godot/godot-3.2.3.ebuild

```diff
@@ -0,0 +1,136 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6..9} )
```

Comment:
You can drop Python 3.6 from this list because Gentoo no longer supports it.

Reply:
python is not used by godot per se, but is a runtime dependency for scons, so it is just having what scons have.

Reply:
That's all right, but Python 3.6 is not longer provided by Gentoo so the ebuild should not have that version listed in the `PYTHON_COMPAT`. Instead, the line should be like this: `PYTHON_COMPAT=( python3_{7..9} )`

Reply:
I suppose it won't harm if I remove it. But python 3.6 is still supported until 2021-12-23 according to the website.

Reply:
You're right, I misspoke because it is still available in the tree -- rather, it looks like it's just [not receiving bug fixes anymore](https://www.gentoo.org/support/news-items/2020-04-22-python3-7.html); regardless, it'll probably be prudent not to have it in new ebuilds. Thanks! :+1: